tests/testdata/result_captions.Rmd

```diff
@@ -46,6 +48,9 @@ verschillend zijn van de schrijfwijze van de tabelhoofding.  En referentie tabel
 1 ook eens splitsen over meerdere regels, en (tabel 2.2) tussen haakjes. En
```

Comment:
should (tabel 2.2) be detected as well? So (tabel \@ref(tab:Tabel2.2)). Probably the regex is case-sensitive?

Reply:
Good point!  It is indeed case-sensitive (could be changed in `gsub()`, but I would keep it as is), and `(tabel 2.2)` was meant to be detected, so I should have written it with a capital...